public/app/plugins/datasource/loki/language_provider.ts

```diff
@@ -120,7 +123,7 @@ export default class LokiLanguageProvider extends LanguageProvider {
     return { suggestions };
   }
 
-  getLabelCompletionItems({ text, wrapperClasses, labelKey, value }: TypeaheadInput): TypeaheadOutput {
+  getLabelCompletionItems({ text, wrapperClasses, labelKey, value }: TypeaheadInput, { range }: any): TypeaheadOutput {
```

Comment:
can't see an update call to this function, where is this called with range passed in?

Reply:
See Line 91:
```javascript
 return this.getLabelCompletionItems.apply(this, arguments);
```
and then in `LokiQueryFieldForm.tsx`:
```javascript
    const result = datasource.languageProvider.provideCompletionItems(
      { text, value, prefix, wrapperClasses, labelKey },
      { history, range }
    );
```